tests/test_mock.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+from unittest.mock import Mock, create_autospec
+
+from pnpq.apt.connection import AptConnection
+from pnpq.apt.protocol import (
+    Address,
+    AptMessage,
+    AptMessage_MGMSG_MOT_GET_USTATUSUPDATE,
+    AptMessage_MGMSG_MOT_MOVE_ABSOLUTE,
+    ChanIdent,
+    Status,
+    StatusBits,
+)
+from pnpq.devices.polarization_controller_thorlabs_mpc320 import (
+    PolarizationControllerThorlabsMPC320,
+)
+from pnpq.units import pnpq_ureg
+
+
+def test_polarization_controller_move_absolute() -> None:
+
+    connection = create_autospec(AptConnection)
+
+    def mock_send_message_expect_reply(
+        sent_message: AptMessage,
+        match_reply_callback: Callable[
+            [
+                AptMessage,
+            ],
+            bool,
+        ],
+    ) -> None:
```

Comment:
I'm somewhat surprised that mypy is OK with this having a different return type than the real function, but this is good enough for now.

Reply:
We don't have any return statement for this function though. Wouldn't that mean it returns `None`?

Reply:
Yeah, but I'm thinking about this from the point of view of a strongly typed language. If we're implementing the interface defined by the `AptConnection` class, then this function has to conform to the same type signature as the real function. But something in here, either in mypy or in autospec or Mock, is causing that to be missed. Oh well.

Reply:
Ohh, yeah. I see what you mean there. That is quite strange.